zeppelin/converter.py

```diff
@@ -143,28 +154,49 @@ def process_results(self, paragraph):
                     output_options[msg['type']](msg)
 
     def parse_date(self, date):
+
+        """
+        Converts string to date object. A sample string with this format
+        is 'Feb 29, 2017 04:39:59 pm'.
+
+        """
+
         return datetime.strptime(date, '%b %d, %Y %I:%M:%S %p')
 
     def process_date_created(self, text):
-        """ Sets date_created to the oldest date """
+
+        """
+        Sets date_created to the oldest date (date created).
+
+        """
         if self.date_created == 'N/A':
             self.date_created = text
         if self.parse_date(text) < self.parse_date(self.date_created):
             self.date_created = text
 
     def process_date_updated(self, text):
-        """ Sets date_updated to the most recent date """
```

Comment:
It's cool to put everything on one line of you have a short description, no need to surround it with blank lines. Actually, if you have a short description, I would suggest to put everything on one line
```python
"""My short description"""
```
instead of 
```python
"""
My short description
"""
```